app/models/meeting.js

```diff
@@ -33,6 +33,11 @@ export default class Meeting extends Model {
     async: true,
   })
   internalDocumentPublicationActivity;
+  @belongsTo('minutes', {
+    inverse: 'isMinutesForMeeting',
```

Comment:
A property/relation starting with 'is', 'has', 'can', etc sounds like a boolean.
perhaps just `minutesForMeeting` ?

Reply:
@tdn thoughts? Goes against your suggestion